src/ui-kit/form-controls/autocomplete-multiselect/autocomplete-multiselect.component.ts

```diff
@@ -977,6 +979,10 @@ export class SamAutocompleteMultiselectComponent
     this.reachedEndOfList(elements,this.searchText);
   }
 
```

Comment:
Add method comment and add unit test for the method

Reply:
added @yerramshilpa can you review + publish a new version of sam-ui-elements?